<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterWindow</class>
 <widget class="QDialog" name="RegisterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Register</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>431</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="reg_labelUsername">
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelRetype">
      <property name="text">
       <string>Re-type Password</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="reg_lineEditUsername">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>25</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="reg_labelUsernameErr">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;* Already Existing&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="reg_labelPass">
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="reg_lineEditPass">
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>25</height>
       </size>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="reg_lineEditRetype">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>25</height>
       </size>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="labelRetypeErr">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;* Not Matching&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBoxDOB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>291</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Date of Birth</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item row="0" column="2">
      <widget class="QLabel" name="labelYear">
       <property name="text">
        <string>Year</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelMonth">
       <property name="text">
        <string>Month</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="labelDay">
       <property name="text">
        <string>Day</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QComboBox" name="comboBoxMonth">
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="currentText">
        <string/>
       </property>
       <property name="maxVisibleItems">
        <number>12</number>
       </property>
       <property name="maxCount">
        <number>11</number>
       </property>
       <property name="duplicatesEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditDay"/>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="lineEditYear"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="labelDOBErr">
   <property name="geometry">
    <rect>
     <x>318</x>
     <y>189</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;* Age is less than 12&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>341</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>30</number>
    </property>
    <item>
     <widget class="QGroupBox" name="groupBoxGender">
      <property name="title">
       <string>Gender</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>131</width>
         <height>52</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="radioButtonMale">
          <property name="text">
           <string>Male</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButtonFemale">
          <property name="text">
           <string>Female</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBoxAccType">
      <property name="title">
       <string>Account Type</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>131</width>
         <height>52</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="radioButtonAdmin">
          <property name="text">
           <string>Admin</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButtonUser">
          <property name="text">
           <string>User</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBoxGenres">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>251</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Favorite Genre(s)</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>231</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="horizontalSpacing">
      <number>50</number>
     </property>
     <item row="0" column="0">
      <widget class="QCheckBox" name="checkBoxAction">
       <property name="text">
        <string>Action</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="checkBoxDrama">
       <property name="text">
        <string>Drama</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="checkBoxComedy">
       <property name="text">
        <string>Comedy</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="checkBoxHorror">
       <property name="text">
        <string>Horror</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="checkBoxRomance">
       <property name="text">
        <string>Romance</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="checkBoxOther">
       <property name="text">
        <string>Other</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="reg_regButton">
      <property name="text">
       <string>Register</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelRegErr">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;* All fields must be filled&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
